ui/apps/platform/src/Containers/NetworkGraph/NetworkGraphPage.tsx

```diff
@@ -42,6 +48,8 @@ const includePorts = true;
 
 function NetworkGraphPage() {
     const [edgeState, setEdgeState] = useState<EdgeState>('active');
+    const [activeModel, setActiveModel] = useState<CustomModel>(emptyModel);
+    const [extraneousFlowsModel, setExtraneousFlowsModel] = useState<CustomModel>(emptyModel);
```

Comment:
Ok for this PR, but I think we need a comment and a Jira task to update this state to be URL-driven, based on the plan in our design doc.

> For controls that cause the front-end to display the data differently, without having to make another API call, we will have to create appropriate param names. Because our Search options are always Title-Case strings with spaces between distinct words, to avoid collisions we should use appropriate camelCase query param names for display-only features, like showConnections=allowed and namespaceFlows=hide
https://docs.google.com/document/d/1QMI0PQ_duKJ_6aqD3SU8gNjBTUDPyNBMOktEG5opABc/edit#heading=h.pan7jow300u7